composer.json

```diff
@@ -7,7 +7,6 @@
         "php": "^8.1",
         "phpstan/phpstan": "^1.9.3",
         "nette/utils": "^3.2",
-        "symfony/finder": "^6.2",
```

Comment:
Could you update the other `composer.json` as well? So we don't require it anywhere